nnunet_scripts/ensemble_inference_and_evaluation.sh

```diff
@@ -0,0 +1,36 @@
+# The first argument should be the path to the datasets.
```

Comment:
```suggestion
# The first argument should be the path to the folder containing the folds to ensemble
```

IIUC this is what you meant? Or maybe I misunderstood the README

Reply:
@hermancollin Actually neither are correct. it should be the nnunet folder path that is provided as the first argument to the script.

Reply:
I fixed it now!